content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -3,14 +3,117 @@ title: Configure the Datadog Tracing Library
 
 {{< partial name="apm/apm-compatibility.html" >}}
 
 <br>
 
 To instrument an application written in a language that does not yet have official library support, see the list of [community tracing libraries][1].
 
+## Common configuration options
+The following configuration options behave consistently across multiple Datadog SDKs:
+
+### Traces
+
+`DD_TRACE_<INTEGRATION>_ENABLED`
+: **Default**: `true` <br>
+**Supported Input**: Boolean <br>
+**Not Supported In**: Go SDK<br>
+**Description**: Enables or disables instrumentation for the specified `<INTEGRATION>`. The integration name must be in uppercase (for example, `DD_TRACE_KAFKA_ENABLED=true`)
+
+`DD_TRACE_RATE_LIMIT`
+: **Default**: `100` <br>
+**Supported Input**: A positive integer<br>
+**Description**: Sets the maximum number of traces to sample per second; applies only when either `DD_TRACE_SAMPLING_RULES` or `DD_TRACE_SAMPLE_RATE` is set.
+
+`DD_TRACE_HEADER_TAGS`
+: **Default**: `null` <br>
+**Supported Input**: A comma-separated string representing a list of case-insensitive HTTP headers, with an optional mapping to a custom tag name. Example: `User-Agent:my-user-agent,Content-Type`. <br>
+**Description**: Automatically apply specified HTTP headers as span tags. If a custom tag name is not specified, the tag key defaults to `http.request.headers.<normalized-header-name>` for request headers and `http.response.headers.<normalized-header-name>` for response headers.
+
+`DD_TRACE_ENABLED`
+: **Default**: `true` <br>
+**Supported Input**: Boolean <br>
+**Description**: Enables or disables the sending of traces to the trace agent.
+
+`DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP`
+: **Default**: <br>
+    ```
+    (?i)(?:(?:"|%22)?)(?:(?:old[-_]?|new[-_]?)?p(?:ass)?w(?:or)?d(?:1|2)?|pass(?:[-_]?phrase)?|secret|(?:api[-_]?|private[-_]?|public[-_]?|access[-_]?|secret[-_]?|app(?:lication)?[-_]?)key(?:[-_]?id)?|token|consumer[-_]?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)(?:(?:\s|%20)*(?:=|%3D)[^&]+|(?:"|%22)(?:\s|%20)*(?::|%3A)(?:\s|%20)*(?:"|%22)(?:%2[^2]|%[^2]|[^"%])+(?:"|%22))|(?:bearer(?:\s|%20)+[a-z0-9._\-]+|token(?::|%3A)[a-z0-9]{13}|gh[opsu]_[0-9a-zA-Z]{36}|ey[I-L](?:[\w=-]|%3D)+\.ey[I-L](?:[\w=-]|%3D)+(?:\.(?:[\w.+/=-]|%3D|%2F|%2B)+)?|-{5}BEGIN(?:[a-z\s]|%20)+PRIVATE(?:\s|%20)KEY-{5}[^\-]+-{5}END(?:[a-z\s]|%20)+PRIVATE(?:\s|%20)KEY(?:-{5})?(?:\n|%0A)?|(?:ssh-(?:rsa|dss)|ecdsa-[a-z0-9]+-[a-z0-9]+)(?:\s|%20|%09)+(?:[a-z0-9/.+]|%2F|%5C|%2B){100,}(?:=|%3D)*(?:(?:\s|%20|%09)+[a-z0-9._-]+)?)
+    ```
+: **Supported Input**: A regex string <br>
+: **Description**: Applies a regex to redact sensitive data from query strings on incoming HTTP requests. Matches are replaced with `<redacted>`. If an empty string is passed, no obfuscation occurs. The resulting value is reported in the `http.url` tag.
+
+### Diagnostics
+
+`DD_TRACE_LOG_DIRECTORY`
+: **Default**: Varies by SDK, environment, and runtime. [Read more here][2] <br>
```

Comment:
The link here doesn't really describe the differences in defaults on first read. Should we update that page as well?